decidim-elections/app/controllers/decidim/elections/admin/answers_controller.rb

```diff
@@ -58,15 +58,31 @@ def update
         end
 
         def select
+          change_selected(false)
```

Comment:
This confuses me a bit, wouldn't be easier to use `change_selected(true)` here (and change the `update_answer_selection!` method on the command)?

Reply:
Yeah, definitely. 😅